src/time_zone_info.cc

```diff
@@ -470,7 +470,7 @@ bool TimeZoneInfo::Load(const std::string& name, ZoneInfoSource* zip) {
   // Determine the before-first-transition type.
   default_transition_type_ = 0;
   if (seen_type_0 && hdr.timecnt != 0) {
-    std::uint_fast8_t index = 0;
+    std::int_fast8_t index = 0;
```

Comment:
I still don't understand why this should be changed.  Indeed, without it (the status quo) -Wsign-compare compiles cleanly, whereas with the change to int_fast8_t I see warnings firing.

So, I'm going to close this PR.  Please followup if you believe there are still issues.  Thanks.